sphinx_gallery/gen_rst.py

```diff
@@ -565,6 +565,13 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
     time_elapsed = 0
     block_vars = {'execute_script': execute_script, 'fig_count': 0,
                   'image_path': image_path_template, 'src_file': src_file}
+
+    argv_orig = sys.argv[:]
+    if block_vars['execute_script']:
+        print('Executing file %s' % src_file)
```

Comment:
We currently use the Sphinx logger not the print statement. The execution of the file is stated in `generate_dir_rst` and not here anymore.